js/components/ImportIndicatorsPopover.js

```diff
@@ -1,157 +1,273 @@
+                                        gettext('There was a server-related problem')
+                                    }    
+                                </p>
+                                <button className="btn btn-sm btn-primary" onClick={() => setViews(INITIAL) }>Try again</button>
+                            </div>
+                        )
+                }
+            })()}      
+        </React.Fragment>
+    )
 }
 
-export class ImportIndicatorsButton extends BootstrapPopoverButton {
-    popoverName = "importIndicators";
-    popoverTitle = gettext("Import indicators");
+const AdvancedImport = () => {
+    const { tierLevelsRows } = useContext(ImportIndicatorsContext);
 
-    constructor(props) {
-        super(props);
-    }
+    const [expanded, setExpanded] = useState(false);
 
-    getPopoverContent = () => {
-        return (
-            <ImportIndicatorsPopover program_id={this.props.program_id}/>
-            );
-    }
-
-    render() {
-        return (
-            <React.Fragment>
-                <button
-                type="button"
-                ref="target"
-                className="btn btn-sm btn-primary pl-2"
-                >
-                    <i className="fas fa-download"></i>
+    return (
+        <div>
+            <div className="advanced-button">
+                <FontAwesomeIcon icon={ expanded ? 'caret-down' : 'caret-right' } /> &nbsp;
+                <a 
+                    data-toggle="collapse" 
+                    href="#optionsForm" 
+                    role="button" 
+                    aria-expanded={ expanded } 
+                    aria-controls="optionsForm" 
+                    onClick={ () => setExpanded(!expanded) }
+                    >
                         {
-                            //  # Translators: a button to open the import indicators menu
-                            gettext('Import indicators')
+                            // # Translators: Click to view the Advanced Option section
+                            gettext('Advanced options')
                         }
-                </button>
-            </React.Fragment>
-        );
+                    </a>
+            </div>
+            <div id="optionsForm" className="collapse">
+                <p className="advanced-text">
+                    {
+                        // # Translators: Details explaining that by default the template will include 10 or 20 rows per result level. You can adjust the number if you need more or less rows.
+                        gettext('By default, the template will include 10 or 20 indicator rows per result level. Adjust the numbers if you need more or fewer rows.')
+                    }
+                </p>
+                {tierLevelsRows.map((level, i) => <AdvancedOptions key={i} level={level} i={i} />)}
+            </div>
+        </div>
+    )
+}
+
+const AdvancedOptions = ({ level, i }) => {
```

Comment:
Ok, once again, super nitpicky/stylistic, feel free to ignore.  When I see the name AdvancedOptions, I think it's going to be a section of various different toggles.  But it's really just a row of similar dropdown fields.  So maybe just rename to something like IndicatorCountOption, or something like that?  The singular would also make it clear that it's a single row of many.

Reply:
I renamed this component to LevelIndicatorCount and updated the classes to match the same logic.